src/server/string_family_test.cc

```diff
@@ -399,4 +399,58 @@ TEST_F(StringFamilyTest, GetDel) {
   ASSERT_THAT(resp, ArgType(RespExpr::NIL));
 }
 
+TEST_F(StringFamilyTest, GetEx) {
+  using std::chrono::duration_cast;
+  using std::chrono::milliseconds;
+  using std::chrono::seconds;
+  using std::chrono::system_clock;
+
+  auto resp = Run({"set", "foo", "bar"});
+  ASSERT_THAT(resp, "OK");
+
+  // resp = Run({"getex", "foo", "EX", "0"});
+  // ASSERT_THAT(resp, ErrArg("invalid expire time"));
+
+  // resp = Run({"getex", "foo", "EX", "-1"});
+  // ASSERT_THAT(resp, ErrArg("invalid expire time"));
+
+  EXPECT_EQ(Run({"getex", "foo"}), "bar");
+
+  auto now = duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count();
```

Comment:
you should not use real clock. Instead use `TEST_current_time_ms` similarly to how it's done in 
`GenericFamilyTest.Expire`.